pages/status.tsx

```diff
@@ -118,14 +134,20 @@ const Status: FC = () => {
           </p>
         </StatusInfo>
       ) : checkStatusReponse === null ? (
-        <StatusInfo
-          handleGoBackClick={handleGoBack}
-          goBackText={t('go-back')}
-          goBackStyle="primary"
-          checkAgainText={t('check-again')}
-        >
-          <p className=" mb-6 text-2xl">{t('unable-to-find-status')}</p>
-        </StatusInfo>
+        <>
+          <LinkSummary
+            id={'noMatchLinkSummary'}
+            links={noMatchHref}
+          ></LinkSummary>
```

Comment:
I think this should come after the status Info, as that would be the primary information the user is looking for.